Tests/winswTests/Util/CLITestHelper.cs

```diff
@@ -0,0 +1,116 @@
+﻿using System;
+using System.IO;
+using JetBrains.Annotations;
+using winsw;
+
+namespace winswTests.Util
+{
+    /// <summary>
+    /// Helper for WinSW CLI testing
+    /// </summary>
+    public static class CLITestHelper
```

Comment:
Nice idea.  to test a main method.  Should we be instead trying to get as much functionality out of Main as possible?  Allowing us to test  each call in a separate internal static method as an example.  We can then use the following attribute to expose to the test project https://msdn.microsoft.com/en-us/library/system.runtime.compilerservices.internalsvisibletoattribute%28v=vs.110%29.aspx